content/en/docs/concepts/architecture/nodes.md

```diff
@@ -377,6 +377,29 @@ For example, if `ShutdownGracePeriod=30s`, and
 for gracefully terminating normal pods, and the last 10 seconds would be
 reserved for terminating [critical pods](/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical).
 
+{{< note >}}
+When pods were evicted during the graceful node shutdown, they are marked as failed.
+Running `kubectl get pods` shows the status of the the evicted pods as `Shutdown` (`Terminated` after version 1.22).
+And `kubectl describe pod` indicates that the pod was evicted because of node shutdown:
+
+```
+Status:         Failed
+Reason:         Shutdown
+Message:        Node is shutting, evicting pods
+```
+
+After 1.22:
```

Comment:
Hi @SergeyKanzhelev .
Is there a good reason to include the `kubectl describe` output and status for Kubernetes version 1.22?

Reply:
do you mean it should go to `dev-1.22` branch? I included it mostly so I wouldn't forget to update it once 1.22 is out. And when 1.22 is out, we likely will want to keep both versions around for some time.

Reply:
Since the v.122 status/source change is not available in v1.21, I would wait to add this comment.

Reply:
reverted